server/server.go

```diff
@@ -645,34 +645,16 @@ func (server HTTPServer) reportEndpoint(writer http.ResponseWriter, request *htt
 		return
 	}
 
-	var rules []proxy_types.RuleWithContentResponse
-	var rulesNotFound []types.RuleID
-	hasNoErrors := true
-
+	rules := []proxy_types.RuleWithContentResponse{}
 	for _, aggregatorRule := range aggregatorResponse.Report {
-		rule, successful, hasNoError := server.fetchRuleContent(aggregatorRule, server.getOSDFlag(request))
-
-		hasNoErrors = hasNoErrors && hasNoError
+		rule, successful, _ := server.fetchRuleContent(aggregatorRule, server.getOSDFlag(request))
```

Comment:
I think you should refactor `fetchRuleContent` as well as it's not needed third parameter anymore

Reply:
actually the refactoring must be deeper (I feel), IMHO should not be part of bug fix. WDYT?

Reply:
thank you @bond95. I will create a separated issue for the refactor

Reply:
@tisnik yep, must be, but removing third parameter is part of this PR, because you start unusing it here, so it should be removed here as well